.azure/azure-pipelines.ci.yml

```diff
@@ -134,6 +134,19 @@ stages:
       tls: openssl
       extraBuildArgs: -DisableLogs -Generator Ninja -ToolchainFile cmake/toolchains/arm-pi-gnueabihf.toolchain.cmake
 
+- stage: build_macos
+  displayName: Build macOS
+  dependsOn: []
+  jobs:
+  # Officially supported configurations.
+  - template: ./templates/build-config-user.yml
+    parameters:
+      image: macOS-10.15
+      platform: macos
+      arch: x64
+      tls: openssl
+      extraBuildArgs: -DisableLogs
```

Comment:
In our CI, I want logging enabled. For local builds, if people don't want to use clog, they can disable.